app/src/main/java/dev/arkbuilders/arkmemo/ui/views/WaveView.kt

```diff
@@ -47,6 +48,7 @@ class WaveView(context: Context, attrs: AttributeSet) : View(context, attrs) {
     }
 
     private fun computeWave(amplitude: Int) {
+        Log.d("WaveView", "computeWave amplitude: $amplitude height: $height")
```

Comment:
Extract tag to constant here as well, it maybe used it other functions in future